src/Console/Command/ProcessCommand.php

```diff
@@ -89,9 +89,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $processResult = $this->processResultFactory->create($systemErrorsAndFileDiffs);
         $outputFormatter->report($processResult, $configuration);
 
-        // invalidate affected files
-        $this->invalidateCacheForChangedAndErroredFiles($processResult);
```

Comment:
is this will work on `NonPhpFileProcessor` as well?

https://github.com/rectorphp/rector-src/blob/0204a2353850c98a38dba36c9a8dd031ae2da0c3/src/NonPhpFile/NonPhpFileProcessor.php#L18